flocker/docs/bootstrap/_extension.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Bootstrap extension entry point.
+"""
+
+from docutils import nodes
+from sphinx.writers.html import HTMLTranslator
+from . import _simple, _tabs
+
+
+def setup(app):
+    """
+    Entry point for sphinx extension.
+    """
+
+    for module in [_simple, _tabs]:
+        module.setup(app)
+
+
+class HTMLWriter(HTMLTranslator):
+    """
+    Overrides part of the default HTMLTranslator to provide specific
+    class names on some generated HTML elements.
+    """
+
+    def visit_table(self, node):
+        self._table_row_index = 0
+        self.context.append(self.compact_p)
+        self.compact_p = True
+        classes = ' '.join(['docutils', 'table', 'table-striped',
+                            self.settings.table_style]).strip()
```

Comment:
Probably, I did this as a very quick hack to make tables come out right. The changes required are that tables come out in the HTML with classes of `table table-striped` on the `table` tag and cell content wrapped in `<p>` tags. If you know how to do this better, please feel free to commit an improvement to this branch :)